docs/releases.md

```diff
@@ -1,23 +1,39 @@
 ## Releases
 
-## Preparation
-
-To **prepare the extension for a new version**, you must:
-
-- update and commit the version in `package.json`
-- update and commit the `CHANGELOG.md`
-- tag, push, and merge to the `master` branch on GitHub
-
-## Packaging
-
-To **build a signed extension .xpi**, you must: commit and push to the
-`production` branch (ideally as a merge commit from `master`) on GitHub.
-
-## Extension Signing
+### Instructions
+
+1. Update "version" in package.json (and package-lock.json)
```

Comment:
Is it important to follow a convention for the `version` string?

Reply:
`npm` imposes some convention on us (i.e. [semver](http://semver.org/)).

We should be clear on what our convention for what denotes `alpha` versus `beta`, though.